wp-content/themes/core/components/comments/comment/Comment_Controller.php

```diff
@@ -120,4 +124,13 @@ public function get_time( $format = 'c' ): string {
 		return date( $format, get_comment_time( 'U' ) );
 	}
 
+	public function get_starting_markup_element(): string {
+		$element = 'li';
+		if ( 'div' === $this->style ) {
+			$element = 'div';
+		}
+
+		return $element;
```

Comment:
Rather than trying to "know" what wrapper tag we're going to use. I wonder if it would make more sense to add a attribute and property for a `COMMENT_TAG` or similar?

Reply:
We wouldn't need this functionality at all if we just allow the correct tag to be passed into the component (defaulting to an `li` or a `div`).

Reply:
I think we should keep the conventions that the [`end_el` uses in the walker](https://github.com/WordPress/WordPress/blob/27522c6f677f022f0e5155db999e0e5fcbd794ac/wp-includes/class-walker-comment.php#L220). If we pass something different, the `end_el` is still going to check if the style arg is a div and write out the closing element depending on it.

Reply:
Eeew. Core Comments code. :grinning: 

Yep, You're right. Let's keep it consistent with core and stick to the `style` const & property.

One bit of compromise, can we call the method, `get_tag()` instead of `get_starting_markup_element()` so that at least _our_ components' method naming stays consistent?